test/rackup/hello.ru

```diff
@@ -1 +1,3 @@
-run lambda { |env| [200, {"Content-Type" => "text/plain"}, ["Hello World"]] }
+hdrs = {'Content-Type'.freeze => 'text/plain'.freeze}.freeze
```

Comment:
Freezing these is a good call - we're here to benchmark Puma's overhead, not the Rack app 👍